packages/smooth_app/lib/cards/product_cards/smooth_product_card_found.dart

```diff
@@ -102,65 +103,31 @@ class SmoothProductCardFound extends StatelessWidget {
                               children: <Widget>[
                                 Flexible(
                                   child: Text(
-                                    product.productName ??
-                                        AppLocalizations.of(context)
-                                            .unknownProductName,
+                                    (product.productName ??
+                                            appLocalizations
+                                                .unknownProductName) +
```

Comment:
It's a bit strange to display "Unknown product" or "Unknown brand" when we don't have the information. What we do on the web site is that we just don't display the field if it is missing. If we don't have anything at all, we display the barcode.